### PR TITLE
change seisio for seisbase

### PR DESCRIPTION
**Pull request by mdenolle**
big change!! Please review carefully

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,13 +1,14 @@
-name = "SeisIO"
-uuid = "b372bb87-02dd-52bb-bcf6-c30dd83fd342"
-authors = ["Joshua P. Jones"]
-version = "1.2.1"
+name = "SeisBase"
+uuid = "8b7dfb90-41a7-4c72-9751-87603311c074"
+authors = ["SeismoJulia Organization"]
+version = "0.0.0"
```
Comment (haakon-e) [inline Project.toml]:
```suggestion
version = "0.1.0"
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ Old version:
 
 ## **Include tests for new code**
 * We expect at least 95% code coverage on each file.
-* Our target code coverage is 99% on both [CodeCov](https://codecov.io/gh/jpjones76/SeisIO.jl) and [Coveralls](https://coveralls.io/github/jpjones76/SeisIO.jl?branch=main). Code coverage has exceeded 97% consistenly since at least June 2019. Please don't break that for us.
+* Our target code coverage is 99% on both [CodeCov](https://codecov.io/gh/jpjones76/SeisBase.jl) and [Coveralls](https://coveralls.io/github/jpjones76/SeisBase.jl?branch=main). Code coverage has exceeded 97% consistenly since at least June 2019. Please don't break that for us.
```
Comment (haakon-e) [inline CONTRIBUTING.md:60]: CodeCov and Coveralls links are incorrect, still referring to `jpjones76`

File: Project.toml [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
 
 [deps]
 Blosc = "a74b3585-a348-5f62-a45c-50e91977d574"
 DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```
Comment (haakon-e) [inline Project.toml]: We shouldn't need Documenter in the top-level Project.toml

File: README.md [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
+<!-- [![codecov](https://codecov.io/gh/jpjones76/SeisBase.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/jpjones76/SeisBase.jl)[![Coverage Status](https://coveralls.io/repos/github/jpjones76/SeisBase.jl/badge.svg?branch=master)](https://coveralls.io/github/jpjones76/SeisBase.jl?branch=master) [![Documentation Status](https://readthedocs.org/projects/SeisBase/badge/?version=latest)](https://SeisBase.readthedocs.io/en/latest/?badge=latest) -->
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 
 A minimalist, platform-agnostic package for univariate geophysical data.
-This version is a follow up from SeisIO.jl (https://github.com/jpjones76/SeisIO.jl). It is intended to be an stable community maintained package.
+This version is a follow up from SeisBase.jl (https://github.com/jpjones76/SeisBase.jl). It is intended to be an stable community maintained package.
```
Comment (haakon-e) [inline README.md]:
```suggestion
This version is a follow up from SeisIO.jl (https://github.com/jpjones76/SeisIO.jl). It is intended to be an stable community maintained package.
```

File: README.md [hunk 5]
```diff
@@ -2,14 +2,14 @@
...
 
-## Installation | [Documentation](http://seisio.readthedocs.org)
-From the Julia prompt, type: `] add SeisIO`; (Backspace); `using SeisIO`
+## Installation | [Documentation](http://SeisBase.readthedocs.org)
```
Comment (haakon-e) [inline README.md:11]: I'll set up docs. I think the easiest to just have them live on github.io, but this will come in another PR.

File: README.md [hunk 6]
```diff
@@ -2,14 +2,14 @@
...
+From the Julia prompt, type: `] add SeisBase`; (Backspace); `using SeisBase`
```
Comment (haakon-e) [inline README.md]:
```suggestion
From the Julia prompt, type: `] add https://github.com/SeismoJulia/SeisBase.jl`; (Backspace); `using SeisBase`
```
this can be changed once the package is published.

File: README.md [hunk 7]
```diff
@@ -40,6 +40,10 @@ sfdir = realpath("../test/SampleFiles/")
 ```
 
 ## Publications | [Changelog](docs/CHANGELOG.md) | [Issues](docs/ISSUES.md)
-Jones, J.P.,  Okubo, K., Clements. T., \& Denolle, M. (2020). SeisIO: a fast, efficient geophysical data architecture for the Julia language. *Seismological Research Letters* doi: https://doi.org/10.1785/0220190295
+Jones, J.P.,  Okubo, K., Clements. T., \& Denolle, M. (2020). SeisBase: a fast, efficient geophysical data architecture for the Julia language. *Seismological Research Letters* doi: https://doi.org/10.1785/0220190295
```
Comment (haakon-e) [inline README.md]:
```suggestion
Jones, J.P.,  Okubo, K., Clements. T., \& Denolle, M. (2020). SeisIO: a fast, efficient geophysical data architecture for the Julia language. *Seismological Research Letters* doi: https://doi.org/10.1785/0220190295
```

File: docs/CHANGELOG.md [hunk 8]
```diff
@@ -2,12 +2,12 @@
 * `writesac` now allows the user to specify SAC file header version (SAC variable NVHDR) using keyword `nvhdr=`. The default is 6. Versions 6 and 7 are supported.
   + Reason for change: SAC won't read a file if NVHDR is greater than expected.
     - Example: SAC v101.x (NVHDR = 6) *will not* read a SAC file created in/for SAC v102.x (NVHDR = 7), even though the two file header versions are effectively interchangeable.
-* `filtfilt!`: breaking combinations of data and filter parameters should now be identical in `DSP.filtfilt` and `SeisIO.filtfilt!`, even on Float32 data.
+* `filtfilt!`: breaking combinations of data and filter parameters should now be identical in `DSP.filtfilt` and `SeisBase.filtfilt!`, even on Float32 data.
   + Implicit feature request from issue #82.
   + This should never force data conversion to Float64. (We already test this.)
   + It's always possible to choose a filter that outputs NaNs, but the two filtering functions should now behave identically in this regard.
 
-# SeisIO v1.2.0 Release: 2021-02-02
```
Comment (haakon-e) [inline docs/CHANGELOG.md:10]: Could do in another PR, but probably best to just delete all changelog contents

File: docs/ReleaseNotes/release_0.3.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# SeisIO v0.3: **Typeageddon!**
+# SeisBase v0.3: **Typeageddon!**
```
Comment (haakon-e) [inline docs/ReleaseNotes/release_0.3.md]: All release_x.x.md files could probably be deleted too.

File: tutorial/Part_1-Basics.ipynb [hunk 10]
```diff
@@ -45,14 +45,14 @@
     "using Pkg\n",
     "Pkg.resolve()\n",
     "if Sys.iswindows() == false\n",
-    "    if get(Pkg.installed(), \"SeisIO\", nothing) == nothing\n",
```
Comment (haakon-e) [inline tutorial/Part_1-Basics.ipynb:48]: TODO: We should remove jupyter notebook files and instead use code blocks in docs

**Review by haakon-e — COMMENTED** [review]
Looks good -- I haven't looked at the tests yet, more substantial changes is probably desired for new PRs.

File: test/runtests.jl [hunk 11]
```diff
@@ -1,13 +1,13 @@
...
 # =====================================================================
 # Setup
 @info("Please allow 20 minutes for all tests to execute.")
-cd(dirname(pathof(SeisIO))*"/../test")
-if isdir("SampleFiles") == false
-  get_svn("https://github.com/jpjones76/SeisIO-TestData/trunk/SampleFiles", "SampleFiles")
-end
+cd(dirname(pathof(SeisBase))*"/../test")
+# if isdir("SampleFiles") == false
+  # get_svn("https://github.com/jpjones76/SeisBase-TestData/trunk/SampleFiles", "SampleFiles")
```
Comment (haakon-e) [inline test/runtests.jl]: tests seem to fail because we don't download files from this repo.

**Comment by haakon-e** [comment]
I think it's fine to merge this PR with docs failing -- better to fix docs in a separate PR.

File: test/runtests.jl (hunk 11 above)
Comment (niyiyu) [reply test/runtests.jl]:
```suggestion
  # get_svn("https://github.com/jpjones76/SeisIO-TestData/trunk/SampleFiles", "SampleFiles")
```

Yes, looks like the test files are still in SeisIO-TestData repo. Thanks for pointing out

**Comment by niyiyu** [comment]
@haakon-e  @mdenolle Thanks for reviewing the codes! I am going to merge this PR. In the mean time, there are more on the todo list:

1. The docs are originally in rst, which I am not sure whether supported by  Documenter. If possible, convert them to `md` with a `make.jl`? Will fix them in another PR.
2. Remove jupyter notebook files in the tutorials and instead use code blocks in docs.
3. I am running the tests local but are getting some failure. Will fix in later commit.
